contracts/liquidity_hub/bonding-manager/src/commands.rs

```diff
@@ -0,0 +1,373 @@
+use cosmwasm_std::{
+    ensure, Addr, BankMsg, Coin, CosmosMsg, Decimal, DepsMut, Env, MessageInfo, Order, Response,
+    StdError, StdResult, Timestamp, Uint128, Uint64,
+};
+use white_whale_std::pool_network::asset;
+
+use white_whale_std::bonding_manager::Bond;
+
+use crate::helpers::validate_growth_rate;
+use crate::queries::{query_claimable, query_weight, MAX_PAGE_LIMIT};
+use crate::state::{
+    update_global_weight, update_local_weight, BOND, CONFIG, EPOCHS, GLOBAL, LAST_CLAIMED_EPOCH,
+    UNBOND,
+};
+use crate::{helpers, ContractError};
+
+/// Bonds the provided asset.
+pub(crate) fn bond(
+    mut deps: DepsMut,
+    timestamp: Timestamp,
+    info: MessageInfo,
+    _env: Env,
+    asset: Coin,
+) -> Result<Response, ContractError> {
+    helpers::validate_funds(&deps, &info, &asset, asset.denom.clone())?;
+
+    // helpers::validate_claimed(&deps, &info)?;
```

Comment:
why are these validations not being done? i.e. `validate_claimed` and `validate_bonding_for_current_epoch `